api/app/routers/hfi_calc.py

```diff
@@ -69,65 +135,162 @@ def extract_selected_stations(request: HFIResultRequest) -> List[int]:
     return stations_codes
 
 
-@router.post("/load", response_model=HFIResultResponse)
-async def load_hfi_result(request: HFILoadResultRequest,
+@router.post("/fire_centre/{fire_centre_id}/{start_date}/planning_area/{planning_area_id}"
+             "/station/{station_code}/enable/{enable}")
+async def toggle_planning_area_station(
+    fire_centre_id: int, start_date: date,
+    planning_area_id: int, station_code: int,
+    enable: bool,
+    response: Response,  # pylint: disable=unused-argument
+    token=Depends(authentication_required)  # pylint: disable=unused-argument
+):
+    """ Enable / disable a station withing a planning area """
+    # TODO: stub - implement!
+    logger.info('/fire_centre/%s/%s/planning_area/%s/station/%s/enable/%s',
+                fire_centre_id, start_date, planning_area_id, station_code, enable)
+    raise NotImplementedError('This function is not yet implemented.')
```

Comment:
Nice. You may have angered the gods of Agile by prematurely partially implementing this function, but I like that it's here to remind us of remaining work and that the endpoint exists already.